### PR TITLE
fix: [CDS-68477]: HF for redirecting global freeze to list view when clicked from audit trail list view

### PR DESCRIPTION
**Pull request by satyam-harness**
### Summary
Original PR https://github.com/harness/harness-core-ui/pull/15767/files

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by pankajsingh-harness — APPROVED** [review]

**Comment by satyam-harness** [comment]
retrigger prettier

**Comment by satyam-harness** [comment]
retrigger jest

**Review by ishantmehtaharness — APPROVED** [review]
looks good satyam

File: package.json [hunk 1]
```diff
@@ -281,4 +281,4 @@
   "jest-junit": {
     "addFileAttribute": "true"
   }
-}
+}
```
Comment (ishantmehtaharness) [inline package.json:284]: what is this change?

**Review by jenil-harness — APPROVED** [review]

**Pull request merged**